sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -160,6 +160,8 @@
     'WriteToBigQuery',
     ]
 
+logger = logging.getLogger(__name__)
```

Comment:
Nit: our style is to call `logging.info()`, `logging.debug()`, etc. in Beam.

Reply:
I disagree but I will comply.

Reply:
There's this issue: https://issues.apache.org/jira/browse/BEAM-3523
I'm not very familiar with the Python logging module, but it seems that the Beam worker logger expects modules to use a single logger, and that's the root one by default:
https://github.com/apache/beam/blob/f9da7d5c7b0bf5e65664d2568e134ce228367b89/sdks/python/apache_beam/runners/worker/logger.py#L108-L109

I think the simplest way forward is to start a discussion on the beam-dev mailing list.